tools/bundle-size-tools/src/utilities/gitCommands.ts

```diff
@@ -9,9 +9,9 @@ import { execSync } from 'child_process';
  * Gets the commit in main that the current branch is based on.
```

Comment:
Comment needs updating
```suggestion
 * Gets the commit in the target branch that the current branch is based on.
```